### PR TITLE
DOCSP-37539 Change Streams

### PR DESCRIPTION
**Pull request by jordan-smith721**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-37539
Staging - https://preview-mongodbjordansmith721.gatsbyjs.io/pymongo/DOCSP-37539-change-streams/read/change-streams/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/read/change-streams.txt [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+   :emphasize-lines: 2,5
+
+   {'_id': {'_data': '...'}, 'operationType': 'update', 'clusterTime': Timestamp(...), 'wallTime': datetime.datetime(...),
+    'fullDocument': {'_id': ObjectId('...'), 'address': {...}, 'borough': 'Queens', 
+    'cuisine': 'Irish', 'grades': [...], 'name': 'Blarney Castle', 'restaurant_id': '40366356'},
+    'ns': {'db': 'sample_restaurants', 'coll': 'restaurants'}, 'documentKey': {'_id': ObjectId('...')},
+    'updateDescription': {'updatedFields': {'cuisine': 'Irish'}, 'removedFields': [], 'truncatedArrays': []}}
+
+To learn more about pre-images and post-images, see the 
+:manual:`pre- and post-images section </changeStreams#change-streams-with-document-pre--and-post-images>`
```
Comment (rustagir) [inline source/read/change-streams.txt:246]: S: use correct title for Change Streams with Document Pre- and Post-Images

**Review by rustagir — APPROVED** [review]
pretty much lgtm save a few comments!

File: source/read/change-streams.txt [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+
+.. _pymongo-change-stream-update:
+
+.. literalinclude:: /includes/read/change-streams.py
+   :start-after: start-update-for-change-stream
+   :end-before: end-update-for-change-stream
+   :language: python
+
+When you update the collection, the change stream application prints the change
+to the shell as it occurs. The change event printed to the shell resembles the
```
Comment (blink1073) [inline source/read/change-streams.txt:68]: "printed to the shell" sounds confusing for a python user.  I'd say something more like "the change event printed to the output".

Comment (jordan-smith721) [reply source/read/change-streams.txt]: Simplified to just say "prints the change as it occurs"

**Review by blink1073 — APPROVED** [review]
LGTM!

**Pull request merged**